backend/tests/Squidex.Domain.Apps.Entities.Tests/Contents/Queries/ContentQueryServiceTests.cs

```diff
@@ -218,6 +218,33 @@ public async Task QueryAll_should_not_return_contents_if_user_has_no_permission(
             Assert.Empty(result);
         }
 
+        [Theory]
+        [InlineData(1, 0, SearchScope.All)]
+        [InlineData(1, 1, SearchScope.All)]
+        [InlineData(0, 1, SearchScope.All)]
+        [InlineData(0, 0, SearchScope.Published)]
+        public async Task QueryAll_should_return_own_user_contents(int isFrontend, int unpublished, SearchScope scope)
```

Comment:
I think what you want to test here is that q.Created is set depending on the user permissions. I don't see it in this test.

Reply:
do you mean I need  test if permission is ReadOwnContent, query must to has filter CreatedBy?